src/connect/components/breadcrumbs/add-new-item-input.tsx

```diff
@@ -1,12 +1,17 @@
 import { Icon } from '@/powerhouse';
-import { useLayoutEffect, useRef, useState } from 'react';
+import {
+    ComponentPropsWithoutRef,
+    useLayoutEffect,
+    useRef,
+    useState,
+} from 'react';
 import { useKeyboard } from 'react-aria';
 import { Button, Input, PressEvent, TextField } from 'react-aria-components';
 import ClickAwayListener from 'react-click-away-listener';
 import { twMerge } from 'tailwind-merge';
 
 export interface AddNewItemInputProps
-    extends Omit<React.HTMLAttributes<HTMLDivElement>, 'onSubmit'> {
+    extends Omit<ComponentPropsWithoutRef<'input'>, 'onSubmit'> {
```

Comment:
newly caught type error caused by using html div element type for an input — 'placeholder' does not exist on type